pkg/apis/policy/v1alpha1/clusterimagepolicy_types.go

```diff
@@ -214,6 +214,14 @@ type MatchResource struct {
 	ResourceSelector *metav1.LabelSelector `json:"selector,omitempty"`
 }
 
+// URL defines all the propierties to fetch a remote policy
+type URL struct {
```

Comment:
So, I don't think we should call this type URL since there's other types that are URLs. I understand it's scoped to this package, but I'd prefer to not call it URL to avoid confusion.

Reply:
I initially called it Remote, then I change it to avoid conflicts with the TrustRoot.Remote. I can call it RemotePolicy :D.